src/screens/settings-security-pin.tsx

```diff
@@ -28,7 +24,7 @@ export const SettingsSecurityPinScreen = () => {
         await wallets.updateWalletsData(pin);
         await app.updatePin(pin);
         hideModal();
-        app.emit('notification', 'PIN code successfully changed');
+        app.emit('notification', getText(I18N.notificationPinChanged));
```

Comment:
maybe we can wrap `app.emit('notification', getText(I18N.notificationPinChanged))` into sendNotification(text)?